src/app/instance.h

```diff
@@ -81,9 +81,7 @@ class Instance {
   void SetupTunnels(bool is_reloading = false);
 
   /// @brief Should remove old tunnels after tunnels config is updated
-  /// TODO(unassigned): not fully implemented
-  void RemoveOldTunnels(
-      std::vector<std::string>& updated_tunnels);
+  void RemoveOldTunnels(std::vector<std::string>& updated_tunnels);
```

Comment:
The commit says `Instance: fix RemoveOldTunnels` but `RemoveOldTunnels()` is still not fully implemented (see definition). Can you please keep the TODO (or fully implement)?